erigon-lib/state/domain_shared.go

```diff
@@ -1012,6 +1013,7 @@ func (sdc *SharedDomainsCommitmentContext) readStorage(plainKey []byte) (enc []b
 }
 
 func (sdc *SharedDomainsCommitmentContext) Account(plainKey []byte) (u *commitment.Update, err error) {
+	commitment.CurrentCommitmentMetrics.LoadAccount++
```

Comment:
well maybe worth to do it outside of the functions - all usages are easy to find